frontend/packages/component-library-styles/CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+# Contributing to Component Library Styles
+
+Thank you for considering contributing to the Component Library Styles! 🎉  
+We welcome all contributions — whether it’s a bug fix, new token, documentation improvement, or something else
+entirely.  
+This guide will help you get started quickly and ensure a smooth contribution process. To get the full picture of the
+project, please refer to the [README.md](./README.md) file.
+
+## Table of Contents
+
+- [🛣️ Contribution Roadmap](#-contribution-roadmap)
+- [🎨 How to Add a New Style or Token](#-how-to-add-a-new-style-or-token)
+- [🚦Production-ready Checklist](#production-ready-checklist)
+- [🎯 Coding Standards](#-coding-standards)
+- [🧪 How to Test Your Styles](#-how-to-test-your-styles)
+- [🔀 Commit Message Guidelines](#-commit-message-guidelines)
+- [📚 Need Help?](#-need-help)
+- [💡 Tips for a Smooth PR Review](#-tips-for-a-smooth-pr-review)
+- [✅ How to Get Your PR Approved Faster](#-how-to-get-your-pr-approved-faster)
+
+## 🛣️ Contribution Roadmap
+
+Here’s a quick guide to the contribution process:
+
+1. **(The most important step!) Create a thread** in the `#ask-design-system` Slack channel about your change.
+
+   - **Get approval** from @markabarrett and/or @moshebaricdo (design team).
+   - For technical details, ask @levadadenys.
+
+2. **Create a separate PR** with `@code-dot-org/component-library-styles` code changes.
+
+   - Make sure the PR follows the [Best Practices](./README.md/#best-practices).
+   - Include relevant updates to tokens, colors, typography, and styles.
+
+3. **Get it approved** by:
+
+   - ✅ @moshebaricdo and/or @markabarrett (design)
+   - ✅ @levadadenys (technical implementation)
```

Comment:
Do you want to add me and Artem to this too? Ideally we could have a GitHub group with anyone who should look at these so it's not dependent on one person. (Like `@design-system-team` or something.)

Reply:
This would also make it more future proof! (But +1 on adding at least Kelby here either way.)

Reply:
I'm adding only @kelbyhawn right now, but with a plan that later we setup codeowner groups for tech and design sides of component-library and make them required reviewers by default with adding more people into codowner groups as more people get more context on design system